requests_kerberos/__init__.py

```diff
@@ -3,23 +3,27 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Requests is an HTTP library, written in Python, for human beings. This library
-adds optional Kerberos/GSSAPI authentication support and supports mutual
+adds optional Kerberos/Spnego (GSSAPI - Linux/SSPI - Windows) authentication support and supports mutual
 authentication. Basic GET usage:
 
     >>> import requests
     >>> from requests_kerberos import HTTPKerberosAuth
     >>> r = requests.get("http://example.org", auth=HTTPKerberosAuth())
 
+    >>> import requests
+    >>> from requests_kerberos import HTTPSpnegoAuth
+    >>> r = requests.get("http://example.org", auth=HTTPSpnegoAuth())
+
 The entire `requests.api` should be supported.
 """
 import logging
 
-from .kerberos_ import HTTPKerberosAuth, REQUIRED, OPTIONAL, DISABLED
+from .kerberos_ import HTTPKerberosAuth, HTTPSpnegoAuth, REQUIRED, OPTIONAL, DISABLED
```

Comment:
I have seen Spnego, SPNego, SPNEGO. I personally do not like HTTPSPNEGOAuth as it is difficult to read.